presto-iceberg/pom.xml

```diff
@@ -213,7 +213,6 @@
         <dependency>
             <groupId>io.prestosql</groupId>
             <artifactId>presto-main</artifactId>
-            <scope>test</scope>
```

Comment:
This dependency is the under `<!-- for testing -->` comment, so it would be better to move.